src/components/VendorDashboard/VendorDashboard.tsx

```diff
@@ -61,8 +63,13 @@ function VendorDashboard() {
         }
         break;
       case 3:
-        const currentWeddings = weddings.filter((a: any) => a.date == Date.now())
-        if(currentWeddings.length) {
+        const currentWeddings = weddings.filter((a: any) => {
```

Comment:
It seems very handy to be able to sort the weddings by date! Glad you found a library to help with the dates